app/views/admin/badge_achievements/index.html.erb

```diff
@@ -38,7 +38,7 @@
         <td class="whitespace-nowrap"><%= badge_achievement.user.id %></td>
         <td><%= badge_achievement.user.username %></td>
         <td><%= badge_achievement.badge.title %></td>
-        <h5>
+        <div>
```

Comment:
Probably minor, not a blocker: I don't think `div` is allowed inside a `tr` (except if inside a `td`), but I'm also not sure a `h5` is allowed inside a `tr` either, and I guess that was working. Maybe the question here is: what does this accomplish here?

Reply:
Nice catch @jaw6 . I did check and the `div` was actually getting ignored. Therefore I have removed it.
<img width="1217" alt="Screenshot 2023-02-02 at 3 27 01 PM" src="https://user-images.githubusercontent.com/9396084/216293823-a6abe95f-f7ef-4470-9ba1-896fce581c66.png">